store/postgres/src/relational/ddl.rs

```diff
@@ -225,31 +225,35 @@ impl Table {
                 ("gist".to_string(), index_expr)
             }
         } else {
-            let index_expr = if column.use_prefix_comparison {
-                match column.column_type {
-                    ColumnType::String => {
-                        format!("left({}, {})", column.name.quoted(), STRING_PREFIX_SIZE)
-                    }
-                    ColumnType::Bytes => format!(
-                        "substring({}, 1, {})",
-                        column.name.quoted(),
-                        BYTE_ARRAY_PREFIX_SIZE
-                    ),
-                    // Handle other types if necessary, or maintain the unreachable statement
-                    _ => unreachable!("only String and Bytes can have arbitrary size"),
+            Self::calculate_index_method_and_expression(column)
+        }
+    }
+
+    pub fn calculate_index_method_and_expression(column: &Column) -> (String, String) {
+        let index_expr = if column.use_prefix_comparison {
+            match column.column_type {
+                ColumnType::String => {
+                    format!("left({}, {})", column.name.quoted(), STRING_PREFIX_SIZE)
                 }
-            } else {
-                column.name.quoted()
-            };
+                ColumnType::Bytes => format!(
+                    "substring({}, 1, {})",
+                    column.name.quoted(),
+                    BYTE_ARRAY_PREFIX_SIZE
+                ),
+                // Handle other types if necessary, or maintain the unreachable statement
+                _ => unreachable!("only String and Bytes can have arbitrary size"),
+            }
+        } else {
+            column.name.quoted()
+        };
 
-            let method = if column.is_list() || column.is_fulltext() {
-                "gin".to_string()
-            } else {
-                "btree".to_string()
-            };
+        let method = if column.is_list() || column.is_fulltext() {
```

Comment:
We abandoned GIN indexes altogether.  Also, having more than one column in an index should not automatically mean non-btree (would be GiST in this case), GiST should be the exception when `block_range` is in the list specified.  The original issue was about `block_range` appearing in the column list despite not having been specified on the `graphman` command line.

Reply:
We abandoned GIN for lists/arrays, but we do still need it for fulltext search, right?

Reply:
Indeed, @incrypto32 please ignore that part.

Reply:
Ok, so for the default index im going with this
<img width="490" alt="Screenshot 2024-04-08 at 7 17 35 PM" src="https://github.com/graphprotocol/graph-node/assets/44740264/b283a8b1-8510-4258-aff7-29191ac51700">